lib/awful/tag.lua

```diff
@@ -41,6 +41,27 @@ data.tags = setmetatable({}, { __mode = 'k' })
 tag.history = {}
 tag.history.limit = 20
 
+-- Default values
+local defaults = {}
```

Comment:
Yes, I request that change on IRC. Sorry about that, but I don't think having 2 places to change the default is a good idea. You can set beautiful variables in rc.lua if you want. I didn't wanted to have to deal with the confusion having an exported default table would ensue.